others/postfix_evaluation.cpp

```diff
@@ -0,0 +1,160 @@
+/**
+ * @file
+ * @brief Evaluation of Postfix Expression
+ * @author [Darshana Sarma](https://github.com/Darshana-Sarma)
+ * @details
+ * Create a stack to store operands (or values).
+ * Scan the given expression and do following for every scanned element.
+ * If the element is a number, push it into the stack
+ * If the element is a operator, pop operands for the operator from stack.
+ * Evaluate the operator and push the result back to the stack
+ * When the expression is ended, the number in the stack is the final answer
+ */
+#include <algorithm>  // for all_of
+#include <array>      // for std::array
+#include <cassert>    // for assert
+#include <iostream>   // for io operations
+#include <string>     // for stof
+
+class Stack {
+ public:
+    std::array<float, 20> stack{};
+    int stackTop = -1;
```

Comment:
Please document these two variables: what do they do, how do they work, etc..
```suggestion
    std::array<float, 20> stack{}; ///< short info of this variable
    int stackTop = -1;             ///< short info of this variable
```